scripts/ci/Dockerfile

```diff
@@ -79,6 +79,7 @@ ENV GIT_SHA=${GIT_SHA}
 ENV APP=${APP}
 ENV BASEPATH=/
 
-RUN mkdir -p /etc/nginx/templates
-ADD scripts/ci/dockerfile-config/nginx/* /etc/nginx/templates
+RUN mkdir -p /etc/nginx/templates && apk add bash
```

Comment:
just `RUN apk add bash` in a separate layer. don't see what it has to do with the creation of this directory :)